src/Application.php

```diff
@@ -155,4 +166,36 @@ public static function loadProjectConfig(?string $project, string $projectsPath)
             Configure::load($project, 'projects');
         }
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function getAuthenticationService(ServerRequestInterface $request, ResponseInterface $response)
```

Comment:
```suggestion
    public function getAuthenticationService(ServerRequestInterface $request, ResponseInterface $response): AuthenticationService
```
Add return type to function